ExportScripts/export_standalone_monolithic_windows.py

```diff
@@ -183,22 +183,24 @@ def process_command(args: list,
 
 
 # EXPORT SCRIPT STARTS HERE!
-
-
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(prog='Exporter for MultiplayerSample on windows',
-                                 description = "Exports O3DE's MultiplayerSample to the desired install directory in project layout...")
-    parser.add_argument('-ps', '--project-path', type=pathlib.Path, required=True, help='Path to the intended O3DE project.')
-    parser.add_argument('-egn', '--engine-path', type=pathlib.Path, required=True, help='Path to the intended O3DE engine copy.')
+                                 description = "Exports O3DE's MultiplayerSample to the desired output directory with release layout. "
+                                                "In order to use this script, the engine and project must be setup and registered beforehand. "
+                                                "See this example on the MPS Github page: "
+                                                "https://github.com/o3de/o3de-multiplayersample/blob/development/README.md#required-step-to-compile")
+    parser.add_argument('-pp', '--project-path', type=pathlib.Path, required=True, help='Path to the intended O3DE project.')
+    parser.add_argument('-ep', '--engine-path', type=pathlib.Path, required=True, help='Path to the intended O3DE engine.')
     parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Release Directory path location.')
-    parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'], help='The CMake build configuration to use when building project binaries. If tool binaries are built with this script, they will use profile mode.')
+    parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'],
+                        help='The CMake build configuration to use when building project binaries. If tool binaries are built with this script, they will use profile mode.')
     parser.add_argument('-ll', '--log-level', default='ERROR',
                         choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
                         help="Set the log level")
     parser.add_argument('-aof', '--archive-output-format',
                         type=str,
                         help="Format of archive to create from the output directory",
-                        choices=["none", "zip", "gzip", "bz2", "xz"], default="none")
+                        choices=["zip", "gzip", "bz2", "xz"], default="zip")
     parser.add_argument('-bnmt', '--build-non-mono-tools', action='store_true')
```

Comment:
Is this on by default now or off?

Reply:
it is still off by default, as there is a separate flag to decide whether to archive or not (`--archive-output`)

Reply:
I mean specifically: '--build-non-mono-tools'